.github/workflows/vale.yml

```diff
@@ -0,0 +1,18 @@
+name: reviewdog
+on: push
```

Comment:
```suggestion
on:
  pull_request:
  push:
```

I'm not sure without `pull_request` that it trigger a test whenever you create the PR for the first time

Reply:
You're right, thanks! 🙏